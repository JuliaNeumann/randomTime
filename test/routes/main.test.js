import { mount } from 'vue-test-utils';
import Main from '../../app/routes/main/main.vue';

describe('Main', () => {
  const wrapper = mount(Main);

  it('renders the correct markup', () => {
    expect(wrapper.contains('h1')).toBe(true);
    expect(wrapper.contains('p')).toBe(true);
  })

});