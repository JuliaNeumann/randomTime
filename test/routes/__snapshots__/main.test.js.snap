// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Main renders the correct markup 1`] = `
<div
  class="uk-container"
  id="app"
>
  <!---->
   
  <h1
    class="uk-heading-divider"
  >
    Welcome to Random Time!
  </h1>
   
  <p
    class="uk-text-lead"
  >
    Select from the menu above what you want to do.
  </p>
</div>
`;
